### PR TITLE
A1

### PR DESCRIPTION
**Pull request by ces-hungtran**
Truy cập vào thư mục react: 
react-training/modules/a1-react-training/src

### CODE REVIEW

File: modules/a1-react-training/src/components/BeneficiaryProfile.js [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+  decreaseStep = () => {
+    this.state.onUpdateData({ step: this.state.step - 1 });
+  };
+
+  render() {
+    return (
+      <Formik
+        initialValues={{
+          hisEmail: getDefaultString(this.state.hisEmail),
```
Comment (ces-ThaoPhan) [inline modules/a1-react-training/src/components/BeneficiaryProfile.js:59]: - just simply `hisEmail: this.state.hisEmail || ''`
- I don't think we need to use possessive pronouns when naming variables. It makes the name longer and hard to follow. Please update.

**Review by ces-ThaoPhan — CHANGES_REQUESTED** [review]

File: modules/a1-react-training/src/AppComponent.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import BeneficiaryProfile from "./components/BeneficiaryProfile";
+import FinishCheck from "./components/FinishCheck";
+import InsuranceProfile from "./components/InsuranceProfile";
+import PreviousButton from "./components/PreviousButton";
+import * as RENDER_STATE from "./constants/RenderState";
+
+class AppComponents extends React.Component {
```
[the remark this answers is not in the document]
Comment (ces-hungtran) [reply modules/a1-react-training/src/AppComponent.js:4]: > Please transfer components to stateless components + react hooks

Why can't we use class component here?
